helpers/disabled_gpu

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+# Miner options use specified devices
+# Bminer    : -devices 0,1,2,3,4,5,6,7,8
+# CCMINER   : --devices 0,1,2,3,4,5,6,7,8
+# CLAYMORE  : -di 0123456789abcdefghi
+# DSTM      : --dev 0 1 2 3 4 5 6 7 8
+# ETHMINER  : --cuda_devices 0 1 2 3 4 5 6 7 8 
+# EWBF      : --cuda_devices 0 1 2 3 4 5 6 7 8
+# GENOIL    : ??? same as ETHMINER ??? skipping for the moment
+# Note: For ETHMINER, you might need to set environmental variable to have cuda_devices sorted by PCI_BUS ID
+#       refer to https://github.com/ethereum-mining/ethminer FAQ: CUDA GPU order changes sometimes. What can I do?
+#
+
+#
+# Provide functions and set global variables required for disabling GPUS in miners
+# PREREQUISITE  : DISABLED_GPUS is set Prior (in 1bash and loaded prior calling this script)
+# OUTCOME 1     : Declare DISABLED_GPU_ARRAY and marked the index of the array as true as defined in DISABLED_GPUS
+#               : E.g. DISABLED_GPUS="1 4"
+#                      DISABLED_GPU_ARRAY[1]=1
+#                      DISABLED_GPU_ARRAY[4]=4
+# OUTCOME 2     : Provide build_miner_devices_opt that will assigned all usable gpus to each miner's extension opt
+#               : E.g. DISABLED_GPUS="1 4"
+#               : BMINER_OPTS+=" -devices 0,2,3,5,6,..."
+#               : CCMINER_OPTS+=" --devices 0,2,3,5,6,..."
+#               : CLAYMORE_OPTS+=" -di02356..."
+#               : DSTM_OPTS+=" --dev 0 2 3 5 6 ..."
+#               : ETHMINER_OPTS+=" --cuda_devices 0 2 3 5 6 ..."
+#               : EWBF_OPTS+=" --cuda_devices 0 2 3 5 6 ..."
+#
+
+if [ -z "$DISABLED_GPUS" ]; then
+    echo "DISABLED_GPUS not set"
+else
+    # Export CUDA_DEVICE_ORDER env variables ??? not sure if this work
+    export CUDA_DEVICE_ORDER=PCI_BUS_ID
```

Comment:
As long as this helper script is included into 0miner by `source` command this variable will be exported to any child process of 0miner, an therefore any miner it launches.